roles/manifest/meta/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+dependencies:
+  - role: common_variables
```

Comment:
I *think* cause remember is hard, we landed on not going the common_variables design route but instead opting to define the common variables through an example vars file e.g. `vars.yaml.example` or some such at the root of the repository. Possibly even documenting them in the README as well.

@evgeni can you weigh in what you recall

Reply:
Yes, it doesn't help that I immediately went out on vacation for most of the week right after this discussion -- I'm struggling to recall as well the exact final design that we landed on. I do recall that it was considered important to have a single file containing common variables to support ease of use.

Reply:
@mdellweg may also recall what we landed on specifically, I just recall we decided not to use a common role and to use this to include an example vars file

Reply:
Right…

So I think we landed on the following "rules":
- if there is no sensible default, don't add it to `defaults`, but document in `README.md` as required (see e.g. https://github.com/theforeman/foreman-ansible-modules/pull/958/files where I did that for the cleanup role)
  - it might be beneficial to have the *common* variables (also?) documented in a more prominent central place, but we don't have such one yet.
- if the variable is only useful for that specific role, prefix it with the role name
- if it's "global" (like username, or organization) no prefix